File: src/main/java/com/neutralplasma/virtusbot/storage/LocaleData.java
package com.neutralplasma.virtusbot.storage;


import java.util.HashMap;

public class LocaleData {
    HashMap<String, String> localedata;

    public LocaleData(HashMap<String, String> localedata){
        this.localedata = localedata;
    }

    public String getLocale(String locale){
        try {
            String data = localedata.get(locale);

            if (data != null) {
                return data;
            }
        }catch (NullPointerException error){
            return null;
        }
        return null;
    }

    public void updateLocale(String locale, String localedata){
        this.localedata.put(locale, localedata);
    }

    public HashMap<String, String> getAllLocales(){
        return this.localedata;
    }
}

File: src/main/java/com/neutralplasma/virtusbot/storage/locale/LocaleData.kt
package com.neutralplasma.virtusbot.storage.locale

import java.util.*

class LocaleData(var allLocales: HashMap<String, String>) {
    fun getLocale(locale: String?): String? {
        try {
            val data = allLocales[locale]
            if (data != null) {
                return data
            }
        } catch (error: NullPointerException) {
            return null
        }
        return null
    }

    fun updateLocale(locale: String, localedata: String) {
        allLocales[locale] = localedata
    }

}